File: app/src/test/java/ai/rotor/rotorvehicle/ui/dash/DashViewModelTest.java
package ai.rotor.rotorvehicle.ui.dash;

import org.junit.Before;
import org.junit.Test;

import static junit.framework.TestCase.assertEquals;

public class DashViewModelTest {

    private DashViewModel testObj;

    @Before
    public void setup() {
        testObj = new DashViewModel();
    }

    @Test
    public void ShouldProperlyConstructViewModel() {
        assertEquals("Dashboard", testObj.getViewTitle());
    }

}

File: app/src/test/java/ai/rotor/rotorvehicle/ui/dash/DashViewModelTest.kt
package ai.rotor.rotorvehicle.ui.dash

import org.junit.Before
import org.junit.Test
import junit.framework.Assert.assertEquals


class DashViewModelTest {


    lateinit var viewModel: DashViewModel

    @Before
    fun setup() {
        viewModel = DashViewModel()
    }

    @Test
    fun `Should construct correctly`() {

        assertEquals("Dashboard", viewModel.viewTitle)
    }

}